src/components/section-pricing.tsx

```diff
@@ -0,0 +1,77 @@
+import { Button } from '@/components/ui/button';
+import { Card } from '@/components/ui/card';
+import type { PricingCardProps } from '@/types/card-types';
+import { Check } from 'lucide-react';
+
+interface PricingSectionProps {
+  plans: Array<{
+    title: string;
+    features: string[];
+    ctaText: string;
+    ctaVariant: 'default' | 'success';
+    ctaLink: string;
+  }>;
+}
+
+export interface PricingPlan {
+  title: string;
+  features: string[];
+  ctaText: string;
+  ctaVariant: 'default' | 'success';
+  ctaLink: string;
+}
+
+// export interface PricingCardProps extends PricingPlan {}
+
+// export interface PricingSectionProps {
+//   plans: PricingPlan[];
+// }
+
+export function PricingCard({
+  title,
+  features,
+  ctaText,
+  ctaLink,
+}: PricingCardProps) {
+  return (
+    <Card className="flex flex-col p-8 transition-all hover:bg-blue-50">
+      <h3 className="text-2xl font-bold text-center mb-8">{title}</h3>
+      <ul className="flex-1 space-y-6 mb-8">
+        {features.map((feature, index) => (
+          // biome-ignore lint/suspicious/noArrayIndexKey: This is needed for the constants
+          <li key={index} className="flex items-start gap-3">
```

Comment:
```suggestion
          <li key={feature.title} className="flex items-start gap-3">
```

this removes the need to have the biome-ignore. You can always also add an id to the features so it becomes `feature.id`